frontend/app/src/views/history/transactions/TransactionContent.vue

```diff
@@ -0,0 +1,362 @@
+        return totalFound;
+      }
+
+      const entryLimit = limit.value;
+      return Math.min(totalFound, entryLimit);
+    });
+
+    const showUpgradeRow = computed(() => {
+      return limit.value <= total.value && limit.value > 0;
+    });
+
+    const { isSectionRefreshing, shouldShowLoadingScreen } =
+      setupStatusChecking();
+
+    const expanded: Ref<EthTransactionEntry[]> = ref([]);
+
+    const options: Ref<PaginationOptions | null> = ref(null);
+    const account: Ref<GeneralAccount | null> = ref(null);
+
+    const updatePayloadHandler = () => {
+      let paginationOptions = {};
+      if (options.value) {
+        options.value = {
+          ...options.value,
+          sortBy:
+            options.value.sortBy.length > 0 ? [options.value.sortBy[0]] : [],
+          sortDesc:
+            options.value.sortDesc.length > 0 ? [options.value.sortDesc[0]] : []
+        };
+
+        const { itemsPerPage, page, sortBy, sortDesc } = options.value;
+        const offset = (page - 1) * itemsPerPage;
+
+        paginationOptions = {
+          limit: itemsPerPage,
+          offset,
+          orderByAttribute: sortBy.length > 0 ? sortBy[0] : 'timestamp',
+          ascending: !sortDesc[0]
+        };
+      }
+
+      let filterAddress = {};
+      if (account.value) {
+        filterAddress = {
+          address: account.value?.address
+        };
+      }
+      const payload: Partial<TransactionRequestPayload> = {
+        ...filterAddress,
+        ...paginationOptions
+      };
+
+      updatePayload(payload);
+    };
+
+    const updatePaginationHandler = (newOptions: PaginationOptions | null) => {
+      options.value = newOptions;
+      updatePayloadHandler();
+    };
+
+    watch(account, () => {
+      let newOptions = null;
+      if (options.value) {
+        newOptions = {
+          ...options.value,
+          page: 1
+        };
+      }
+
+      updatePaginationHandler(newOptions);
+    });
+
+    const getId = (item: EthTransactionEntry) => item.identifier;
+
+    const selectionMode = setupSelectionMode<EthTransactionEntry>(
+      transactions,
+      getId
+    );
+
+    const gasFee = (tx: EthTransaction) =>
+      toUnit(tx.gasPrice.multipliedBy(tx.gasUsed), Unit.ETH);
+    const toEth = (value: BigNumber) => toUnit(value, Unit.ETH);
+
+    return {
+      account,
+      tableHeaders,
+      transactions,
+      limit,
+      found,
+      total,
+      itemLength,
+      fetch,
+      showUpgradeRow,
+      updatePayload,
+      loading: shouldShowLoadingScreen(Section.TX),
+      refreshing: isSectionRefreshing(Section.TX),
+      expanded,
+      options,
+      gasFee,
+      toEth,
+      updatePaginationHandler,
+      ...selectionMode,
+      ...setupIgnore(
+        IgnoreActionType.ETH_TRANSACTIONS,
+        selectionMode.selected,
+        transactions,
+        fetch,
+        getId
+      )
+    };
+  }
+});
+</script>
+
+<style module lang="scss">
+.table {
+  ::v-deep {
```

Comment:
here it should work fine, but if you want to override vuetify classes in a css module `::v-deep` would not work since it would also prefix them. In that case you can use the `:global` selector instead

Reply:
I think scoped would be better? @kelsos